DecipherNow/helm-charts
# default to installing local charts

Resolves #262 
Resolves #442 

## Changes
- Change requirements.yaml to point to local charts
- Delete all other requirements-* files
- Update the minikube.sh script with dev/prod flags and add make commands
- Update circleci config
- Update docs

## To test

- Make a change to a local helm chart and then run `make dev`. Verify that local charts were installed and you can see your change.
- Run `make destroy` and then `make prod` and verify that hosted charts from nexus were installed.
